org.envirocar.app/res/layout/fragment_dashboard_view_new.xml

```diff
@@ -46,11 +46,26 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent">
 
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginTop="5dp">
+        <ImageView
+            android:id="@+id/fragment_dashboard_obd_help"
+            android:layout_width="30dp"
+            android:layout_height="30dp"
+            android:layout_gravity="left"
+            android:layout_marginStart="10dp"
+            android:tint="@color/white"
+            android:scaleType="fitCenter"
+            android:src="@drawable/others_help" />
+
         <ImageView
-            android:layout_width="wrap_content"
+            android:layout_width="match_parent"
```

Comment:
Can you please say why the ImageView has been changed from `wrap_content` to `match_parent`?

Reply:
You can see the RelativeLayout parent i.e Linear Layout and it's width is match parent so the child layout fits the parent layout according to the defined parameters. As you can see I have hard coded the width and height of the ImageView for obd help so the other (enviroCar logo) doesn't get squeezed I have changed it to match parent.

Reply:
Ok, sounds good.